src/Sentry/SentryMonitorOptions.cs

```diff
@@ -37,6 +42,11 @@ public void Interval(string cronTab)
     /// <param name="unit"></param>
     public void Interval(int interval, MeasurementUnit.Duration unit)
     {
+        if (_type is not SentryMonitorScheduleType.None)
+        {
+            throw new ArgumentException("You tried to set the interval twice. The Check-Ins interval is supposed to be set only once.");
+        }
+
         _type = SentryMonitorScheduleType.Interval;
         _interval = interval;
```

Comment:
Similar to the `cronTab` argument to the overload above, we should either check that the interval + unit here are consistent with the accepted values in the Sentry protocol. Again, although we need to serialize this as a string in order to communicate it to Sentry, internally we could have a more structured representation of this, if that made it easier for the user to express and for our SDK to validate.